rust/example-channels/src/main.rs

```diff
@@ -0,0 +1,71 @@
+use std::thread;
+use std::sync::mpsc::channel;
+
+fn send_receive() {
+    // create channel
+    let (sender, receiver) = channel();
+
+    // thread to send data
+    let sending_thread = thread::spawn(move || {
+        sender.send("important data")
+            .expect("Could not send data")
+    });
+
+    // thread to receive data
+    let receiving_thread = thread::spawn(move || {
+        let data = receiver.recv()
+            .expect("Did not receive data");
+
+        println!("Received: {}", data)
+    });
+
+    // cleanup
+    sending_thread.join().expect("Send has panicked");
+    receiving_thread.join().expect("Receive has panicked");
+}
+
+struct Data {
+    field: String,
+}
+
+fn channel_data_ownership() {
+    // create channel for Data
+    let (sender, receiver) = channel::<Data>();
+
+    // the move keyword allows the thread to take ownership of "sender"
+    let thread1 = thread::spawn(move || {
+        // this is owned by thread 1 here
```

Comment:
```suggestion
        // "data" is owned by thread 1 here
```
Präziser, was "this" ist.